kyrios.sh

```diff
@@ -25,7 +23,7 @@ else
     elif (gte "${python_version}" "Python 2.8"); then
```

Comment:
Future: since Catalina now has python 3, and you've tested this with the version from catalina, should we change this to "Python 3.{whatever ships w/ Catalina}" ?

Reply:
Ack, Catalina ships with 3.7.3. Will make this change.

Reply:
Should we depreciate python versions less than 3? Since python 2 isn't maintained anymore: https://pythonclock.org/.

Reply:
We definitely don't want to be using Python 2.* for any new code we write, and should opportunistically refactor anything we can to 3, but we have tons of legacy code that will take a lot of effort to get up to date. We can't really enforce it too strongly at this point.